tests/foreman/sys/test_katello_certs_check.py

```diff
@@ -191,43 +197,91 @@ def test_positive_update_katello_certs(self):
                 assert result.return_code == 0, 'Not all services are running'
 
     @destructive
-    @stubbed()
-    def test_positive_update_capsule_certs_using_absolute_path(self):
-        """Update certificates on a currently running internal capsule.
+    def test_positive_generate_capsule_certs_using_absolute_path(self):
+        """Create Capsule certs using absolute paths.
 
         :id: 72024757-be6f-49f0-8b88-c57c83f5e7e9
 
         :steps:
 
-            1. Generate custom certs
-            2. Run katello-certs-check with the required valid arguments
-            3. Run  capsule-certs-generate with custom certs and absolute path
+            1. Use Certificates generated in class setup
+            2. Run capsule-certs-generate with custom certs and absolute path
                for --certs-tar
+            3. Assert the certs tar file was created.
 
-        :expectedresults: Katello-certs should be updated.
+        :expectedresults: Capsule certs are generated.
 
-        :CaseAutomation: notautomated
+        :BZ: 1466688
+
+        :CaseAutomation: automated
         """
+        with get_connection(timeout=300) as connection:
+            connection.run('mkdir -p /root/capsule_cert')
+            connection.run(
+                'cp "{0}" /root/capsule_cert/ca_cert_bundle.pem'.format(self.ca_bundle_file_name)
+            )
+            connection.run(
+                'cp "{0}" /root/capsule_cert/capsule_cert.pem'.format(self.cert_file_name)
+            )
+            connection.run(
+                'cp "{0}" /root/capsule_cert/capsule_cert_key.pem'.format(self.key_file_name)
+            )
+            connection.run(
+                'capsule-certs-generate '
+                '--foreman-proxy-fqdn {} '
+                '--certs-tar /root/capsule_cert/capsule_certs_Abs.tar '
+                '--server-cert /root/capsule_cert/capsule_cert.pem '
+                '--server-key /root/capsule_cert/capsule_cert_key.pem '
+                '--server-ca-cert /root/capsule_cert/ca_cert_bundle.pem '
+                '--certs-update-server'.format(self.capsule_hostname),
+                timeout=80,
+            )
```

Comment:
Maybe we should do 
```
restult =  connection.run(
...
self.assertEqual(result.return_code, 0)
```
to cover a minor possibility of the command failing after creating the certs.tar

Reply:
@pondrejk , sure, can add another assert,  but self.assertEqual is form untitest IIRC, so I will use: result.return_code == 0, 'blah'